pump/storage/storage.go

```diff
@@ -732,26 +732,26 @@ func (a *Append) writeToValueLog(reqs chan *request) chan *request {
 		var bufReqs []*request
 		var size int
 
-		write := func(reqs []*request) {
-			if len(reqs) == 0 {
+		write := func(batch []*request) {
```

Comment:
the value's type is request, maybe reqs is better, I think don't need change value name

Reply:
The name `reqs` is already taken above for the channel.